UI/EmuScreen.cpp

```diff
@@ -192,6 +193,10 @@ void EmuScreen::bootGame(const std::string &filename) {
 		g_Config.loadGameConfig(info->id);
 		// Reset views in case controls are in a different place.
 		RecreateViews();
+
+		g_Discord.SetPresenceGame(info->GetTitle().c_str());
+	} else {
+		g_Discord.SetPresenceGame("(unknown)");
```

Comment:
Translate?  Maybe "unknown game" or "Untitled PSP Game" or something?

-[Unknown]

Reply:
Did "Untitled PSP Game". Not sure which translation category makes the most sense though?

Reply:
Screen sounds okay to me.  I guess we could create a separate one for Discord and eventual cloud saves or whatever, but doesn't seem that important...

-[Unknown]